InfluxDB: Interpolate variables based on their type (#75527)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

Should be set for user-assigned identity and should be empty for system-assigned identity.

### workload_identity_enabled

Specifies whether Azure AD Workload Identity authentication should be enabled in datasources that support it.

> For more documentation on Azure AD Workload Identity, review [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/) documentation.

Reviewer: ```suggestion
For more documentation on Azure AD Workload Identity, refer to [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/).
```

> Disabled by default, needs to be explicitly enabled.

Reviewer: ```suggestion
Disabled by default, it needs to be explicitly enabled.
```

### workload_identity_tenant_id

Tenant ID of the Azure AD Workload Identity.

> Allows to override default tenant ID of the Azure AD identity associated with the Kubernetes service account.

Reviewer: ```suggestion
Allows you to override the default tenant ID of the Azure AD identity associated with the Kubernetes service account.
```

### workload_identity_client_id

Client ID of the Azure AD Workload Identity.

> Allows to override default client ID of the Azure AD identity associated with the Kubernetes service account.

Reviewer: ```suggestion
Allows you to override the default client ID of the Azure AD identity associated with the Kubernetes service account.
```

### workload_identity_token_file

Custom path to token file for the Azure AD Workload Identity.

> Allows to set a custom path to the projected service account token file.

Reviewer: ```suggestion
Allows you to set a custom path to the projected service account token file.
```
